tests/tee/attestation/05-teardown.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: kubectl delete secret trustee-secret -n trustee-operator-system
```

Comment:
Question:
Since all the K8s objects are created in the same namespace, won't kuttl cleanup by deleting the namespace and subsequently all the resources in the namespace ?

Reply:
In the existing cluster scenario, the operator is already deployed and configured. So we don't want kuttl to remove the namespace